ADAL/src/ADOAuth2Constants.m

```diff
@@ -94,4 +94,6 @@
 
 //application constants
 NSString* const ADAL_BROKER_SCHEME = @"msauth";
+NSString* const ADAL_BROKER_APP_REDIRECT_URI = @"urn:ietf:wg:oauth:2.0:oob";
```

Comment:
Reply URL doesn't need to be unique. This particular code path will only ever work if you're in the broker app anyways, otherwise you'd have no way to get the response back. :)